app/assets/stylesheets/application.scss

```diff
@@ -11,3 +11,11 @@
 // Your CSS partials
 @import "components/index";
 @import "pages/index";
+
+body {
```

Comment:
Just to be sure, I'd use a class here instead to prevent any issues down the line.